sonaranalyzer-dotnet/src/SonarAnalyzer.DummyScanner/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace DummyScanner
+{
+    class Program
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1118)